Lesson33/Deniz/lesson33-app/src/Exercise4/index.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from "react";
+
+export const Exercise4 = () => {
+  const [items] = useState(["Apple", "Pear", "Banana", "Watermelon"]);
+  const [message, setMessage] = useState("");
+
+  const handleClick = (item) => {
+    if (message.length > 0) {
+      setMessage("");
+    } else {
+      setMessage("Clicked!");
+      console.log("clicked");
+    }
+  };
+
+  return (
+    <div>
+      <ul>
+        {items.map((item, index) => (
+          <li key={index} onClick={() => handleClick(item)}>
```

Comment:
When working with lists in react, avoid using index (number of the element in the array) as a key. Key should be stabile for a list to work correctly. In would be better to use a unique identifier or you can use item directly in this case since the data is array of different strings.